### PR TITLE
added logical_and to ivy/functional/frontends/tensorflow/raw_ops.py

### LINKED ISSUE #14996: logical_and

### PR DESCRIPTION
**Pull request by MuhammadAbuzar36**
Close #14996

### CODE REVIEW

**Comment by MuhammadAbuzar36** [comment]
Close #14996

File: ivy/functional/frontends/tensorflow/raw_ops.py [hunk 1]
```diff
@@ -286,6 +286,9 @@ def Log(*, x, name="Log"):
 LogicalOr = to_ivy_arrays_and_back(map_raw_ops_alias(tf_frontend.math.logical_or))
 
 
+LogicalAnd = to_ivy_arrays_and_back(map_raw_ops_alias(tf_frontend.math.logical_and))
```
Comment (hmahmood24) [inline ivy/functional/frontends/tensorflow/raw_ops.py:289]: Please add the corresponding test as well

**Review by hmahmood24 — CHANGES_REQUESTED** [review]

**Comment by MuhammadAbuzar36** [comment]
@hmahmood24 Thank You for reviewing my PR. I have added the corresponding test function as asked by you and my new commit passes all the tests except one, can you guide me with this one particular test failing?
Thank You again

**Comment by hmahmood24** [comment]
> @hmahmood24 Thank You for reviewing my PR. I have added the corresponding test function as asked by you and my new commit passes all the tests except one, can you guide me with this one particular test failing? Thank You again

Hey @MuhammadAbuzar36. Thanks for making the changes. For which backend is the test failing right now? Looking at the CI it seems to be passing for all backends except `paddle` but it seems that it didn't even run the test against `paddle` to begin with. Can you kindly run the test locally and attach the output here? Make sure you run against all backends by using the flag `--backend "all"`

**Comment by ivy-seed** [comment]
This PR has been labelled as stale because it has been inactive for more than 7 days. If you would like to continue working on this PR, then please add another comment or this PR will be closed in 7 days.

**Comment by hmahmood24** [comment]
This PR has been closed because it has been marked as stale for more than 7 days with no activity.

**Pull request closed without merging**